lamin_cli/_migration.py

```diff
@@ -11,20 +11,20 @@
 
 @click.group()
 def migrate():
-    """Manage database schema migrations."""
+    """Manage sequential database schema migrations."""
 
 
 @migrate.command("create")
 def create():
-    """Create a new migration."""
+    """Create a new numbered migration file."""
     from lamindb_setup._migrate import migrate
 
     return migrate.create()
 
 
 @migrate.command("deploy")
 def deploy():
-    """Deploy migrations."""
+    """Deploy pending migrations to bring database schema up to date."""
```

Comment:
This is a bit confusing IMO -- 'pending' isn't the right term, 'database schema' is also what you have in your code

I'd say "Deploy migrations to the database".